docs/source/introduction.rst

```diff
@@ -1,2 +1,127 @@
 Introduction
 ============
+
+YAGIT (Yet Another Gamma Index Tool) is an efficient open-source library for calculating the gamma index.
+It can be used for the comparison of two images (a reference image and an evaluated image) containing
+2D or 3D radiation dose distributions. It is written in C++ and works on Linux and Windows.
+The code is available on GitHub: `<https://github.com/DataMedSci/yagit>`_.
```

Comment:
```suggestion
YAGIT (Yet Another Gamma Index Tool) is an efficient library for calculating the gamma index.
It can be used for the comparison of two images (a reference image and an evaluated image) containing
2D or 3D radiation dose distributions. It is written in C++ and works on Linux and Windows.
The code is free and open-source. You can find it on GitHub: `<https://github.com/DataMedSci/yagit>`_.
```